javascript/new-src/src/factories/ola.js

```diff
@@ -118,6 +134,7 @@ ola.factory('$ola', ['$http', '$window', 'OLA',
             }
           })
             .then(function(response) {
+              updatehighestChannelNumberUsed(response.data.dmx);
```

Comment:
I forget the exact workflow here, but all updates to the faders come via this function do they? So both local and remote stuff is covered? Ideally this would just be called to deal with the local fader/keypad stuff to let OLAD deal with LTP merges.

Reply:
I saw that the faders' data is updated by polling get_dmx. Remote updates are handled by the backend code. I think get_dmx will return the correctly updated/merged content.

Reply:
Ah yeah you should be fine then, I just wasn't sure if there were a few different paths on the JS side of things. The backend will indeed deal with merging a remote update.

Although that might mean that bringing up a high numbered channel in e.g. the ola_dmxconsole will ripple into the JS faders too, where it perhaps didn't before, but I can't think of an easy solution to that.